rectools/models/implicit_als.py

```diff
@@ -90,6 +90,25 @@ def _fit(self, dataset: Dataset) -> None:  # type: ignore
                 self.verbose,
             )
 
+    def _fit_partial(self, dataset: Dataset) -> None:
+        # deepcopy does not copy model.item_factors and model.user_factors.
```

Comment:
If you call `fit_partial` now on just initialised model, you'll get an error on the last rows of this method since `self.model` is not defined. What is the expected behavior in this case? Let's either support calling `fit_partial` without calling `fit` before or raise a reasonable error

Reply:
Raising an error explicitly ea33c05